Marlin/Configuration.h

```diff
@@ -62,7 +62,7 @@ Here are some standard links for getting your machine calibrated:
 
 // This determines the communication speed of the printer
 // :[2400,9600,19200,38400,57600,115200,250000]
-#define BAUDRATE 250000
+#define BAUDRATE 115200
```

Comment:
This one is was meant for all because of the change made in stepper .cpp but is not related to TWI:
(these PR's were mixed due to lack of skills with Github PR issues)
+      #if BAUDRATE > 115200
+        customizedSerial.checkRx(); // This is probably an old hack that is needed if Baudrate is high due to USART buffer limit
+      #endif
I would be happy to remove that line in stepper but made it conditional to be compatible yet.

There is no need to run serial at 250000 (verified) which can overflow serial input because stepper ISR is too slow (big) and causes overrun in USART (verified), this hack was probably introduced to solve this but is not a neat trick :-(. So answer is 115200 is enough but due to the conditional 250000 would still work. in the long run this issue should be solve by reducing stepper ISR.